storage/reads/array_cursor.gen.go.tmpl

```diff
@@ -0,0 +1,588 @@
+package reads
+
+import (
+	"errors"
+	"fmt"
+	"math"
+
+    "github.com/influxdata/flux/interval"
+    "github.com/influxdata/flux/values"
+	errors2 "github.com/influxdata/influxdb/kit/platform/errors"
+	"github.com/influxdata/influxdb/tsdb/cursors"
+)
+
+const (
+	// MaxPointsPerBlock is the maximum number of points in an encoded
+	// block in a TSM file. It should match the value in the tsm1
+	// package, but we don't want to import it.
+	MaxPointsPerBlock = 1000
```

Comment:
Interestingly we have a MaxPointsPerBlock in the Engine struct in tsm1, but we don't use it, we only use DefaultMaxPointsPerBlock